shared/model/app_stack.go

```diff
@@ -1,27 +1,71 @@
 package model
 
+import (
+	"strings"
+
+	"github.com/chanzuckerberg/happy/shared/config"
+	"github.com/chanzuckerberg/happy/shared/k8s"
+	"github.com/chanzuckerberg/happy/shared/util"
+)
+
 type AppStack struct {
 	CommonDBFields
 	AppMetadata // TODO: might want to change this to AppStackPayload but going with minimal columns for now
 }
 
+type StackMetadata struct {
+	Name               string            `json:"name,omitempty"`
+	Owner              string            `json:"owner,omitempty"`
+	Tag                string            `json:"tag,omitempty"`
+	Status             string            `json:"status,omitempty"`
+	LastUpdated        string            `json:"last_updated,omitempty"`
+	Message            string            `json:"message,omitempty"`
+	Outputs            map[string]string `json:"outputs,omitempty"`
+	Endpoints          map[string]string `json:"endpoints,omitempty"`
+	App                string            `json:"app,omitempty"`
```

Comment:
We will have a duplicate field from this as well because `AppMetadata` has the app name too, maybe we can find a way to consolidate